config/sync/registration_validation.settings.yml

```diff
@@ -0,0 +1,7 @@
+domains: ''
+usernames: "admin\r\nroot\r\nwebmaster\r\nwebadmin\r\npostmaster\r\nadministration\r\nadministrator"
```

Comment:
The user name "siteadmin" would be accepted, please use a wildcard on the word admin and any others you deem necessary

Reply:
done